app/ide-desktop/client/tasks/signArchivesMacOs.ts

```diff
@@ -120,6 +120,14 @@ async function ensoPackageSignables(resourcesDir: string): Promise<Signable[]> {
       `lib/Standard/Google_Api/*/polyglot/java/conscrypt-openjdk-uber-*.jar`,
       ['META-INF/native/libconscrypt_openjdk_jni-osx-*.dylib'],
     ],
+    [
+      `lib/Standard/Tableau/*/polyglot/java/jna-*.jar`,
```

Comment:
(optional) i guess these should all be `'normal strings'` rather than `` `template literals` `` just for consistency